roles/dnsmasq/templates/01-kube-dns.conf.j2

```diff
@@ -11,6 +11,11 @@ server=/{{ dns_domain }}/{{ skydns_server }}
 local=/{{ bogus_domains }}
 
 #Set upstream dns servers
+{% if dnsmask_upstream_dns_servers|length > 0 %}
+{% for srv in dnsmask_upstream_dns_servers %}
+server={{ srv }}
+{% endfor %}
+{% else %}
```

Comment:
why are you disabling default upstream config? it can be separate and not overridden by your custom var